src/main/java/nextstep/courses/domain/SessionEnrolmentInfo.java

```diff
@@ -0,0 +1,59 @@
+package nextstep.courses.domain;
+
+import nextstep.courses.CannotRecruitException;
+import nextstep.users.domain.NsUser;
+
+public class SessionEnrolmentInfo {
```

Comment:
```suggestion
public class SessionEnrolment {
```

[의미가 불분명한 불용어 사용을 자제하라](https://ostarblog.tistory.com/6)
참고해주세요~

Reply:
넵 클린코드에 나왔던 내용인데...습관이란게 무섭네요 ㅠㅠ https://github.com/next-step/java-lms/pull/238/commits/d2dbb64b188cf0494d649b17aeffc0563ced14f1 반영했습니다!